winkmem/README.md

```diff
@@ -0,0 +1,47 @@
+# Windows Kernel Memory Integration
+
+## Overview
+
+Get Windows kernel memory usage to create visualizations and monitors in Datadog.
+
+**Note:** The list of metrics collected by this integration may change between minor Agent versions. Such changes may not be mentioned in the Agent's changelog.
+
+## Setup
+
+### Installation
+
+The Windows Kernel Memory integration is included in the [Datadog Agent][1] package, so you don't need to install anything else on your servers.
+
+### Configuration
+
+1. Edit the `winkmem.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][2]. See the [sample winkmem.d/conf.yaml.example][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `winkmem` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+The Windows Kernel Memory integration collects the metrics defined in [`conf.yaml.example`][3].
```

Comment:
To generate a table of metrics (linking to the conf.yaml.example does not create a table), you have to use the following sentence:

 ```suggestion
See [metadata.csv][6] for a list of metrics provided by this integration.
```

Reply:
Alternatively, you can keep the link to the conf.yaml.example. Either way works!